capmvm/kernel/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM weaveworks/capmvm-kernel-builder:dev AS builder
```

Comment:
In the make file i can override the registry for the builder image but here we are hardcoding to `weaveworks`. The registry should probably be an ARG that we can pass in.